site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -17,6 +17,8 @@ const Planner: FC = () => {
   const isFirstRenderer = useFirstRender();
   const data = useAppSelector(selectYearPlans);
   const transfers = useAppSelector(state => state.roadmap.transfers);
+  
+  const [missingCourses, setMissingCourses] = useState(new Set<string>);
```

Comment:
Also please use `missingPrerequisites` here instead, just for some extra clarity